src/Executables/Atlas/1. Software/Install Software.ps1

```diff
@@ -58,23 +58,23 @@ $Label.Size = New-Object System.Drawing.Size(255, 15)
 $Label.Text = "Download and install software using Chocolatey:"
 $Form.Controls.Add($Label)
 
-# https://community.chocolatey.org/packages/ungoogled-chromium
-$Form.Controls.Add((generate_checkbox "ungoogled-chromium" "ungoogled-chromium"))
+# https://community.chocolatey.org/packages/GoogleChrome
+$Form.Controls.Add((generate_checkbox "Google Chrome" "googlechrome"))
 
 # https://community.chocolatey.org/packages/Firefox
 $Form.Controls.Add((generate_checkbox "Mozilla Firefox" "firefox"))
 
+# https://community.chocolatey.org/packages/microsoft-edge
+$Form.Controls.Add((generate_checkbox "Microsoft Edge" "microsoft-edge"))
```

Comment:
Edge sometimes apparently reverts things according to the AME team, so I would probably *not* include it here just in case. It's most likely that you won't be able to uninstall it after as well, so we probably shouldn't support or recommend it here, at least for now.